.github/workflows/spellcheck_docs.yml

```diff
@@ -0,0 +1,19 @@
+name: spellcheck docs
+
+on:
+pull_request:
+    branches:
+    - main
+    paths:
+    - 'docs/**.md'
```

Comment:
@MKletz Why limit this to MD files? Can it handle comments in code?

Reply:
It only mentioned python and HTML so I wasn't sure if it would identify comments in powershell. We can try